### PR TITLE
Bump consensus version bounds

### PR DESCRIPTION
**Pull request by bolt12**
# Changelog

```yaml
- description: |
    Bump consensus packages version bounds
# uncomment types applicable to the change:
  type:
  # - feature        # introduces a new feature
  # - breaking       # the API has changed in a breaking way
  # - compatible     # the API has changed but is non-breaking
  # - optimisation   # measurable performance improvements
  # - improvement    # QoL changes e.g. refactoring
  # - bugfix         # fixes a defect
  # - test           # fixes/modifies tests
  # - maintenance    # not directly related to the code
  - release        # related to a new release preparation
  # - documentation  # change in code docs, haddocks...
```

# Context

For node 8.9.1

### CODE REVIEW

**Comment by bolt12** [comment]
I think this can be just a revision on CHaP

**Comment by bolt12** [comment]
CHaP release: https://github.com/IntersectMBO/cardano-haskell-packages/pull/704

File: cardano-api/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog for cardano-api
 
+## 8.39.3.0
```
Comment (carbolymer) [inline cardano-api/CHANGELOG.md:3]: Why two version bumps?

File: cardano-api/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+
+- Bump dependencies version bounds
```
Comment (carbolymer) [inline cardano-api/CHANGELOG.md:5]: Can you specify what was bumped here?

Comment (bolt12) [reply cardano-api/CHANGELOG.md:5]: `ouroboros-consensus-diffusion ^>= 0.12`

File: cardano-api/CHANGELOG.md (hunk 1 above)
Comment (bolt12) [reply cardano-api/CHANGELOG.md:3]: That's how Github shows the diff. Because this PR is based on 8.39.2. So it shows a weird diff.

**Comment by carbolymer** [comment]
Can you make another PR which branches off latest `main` which just contains:
* Updated changelog - we would like to keep the log of all API releases
* Dependecies' and indices' bump
* Code changes
?
I think this PR should be closed, because it bumps the API version to a lower than the one in `main` and we don't want that.

**Comment by bolt12** [comment]
@carbolymer yes I can do that. This is something I needed for 8.9.1 which uses cardano-api 8.39.2 . I didn't want to integrate/release latest cardano-api for 8.9.1, that will be done for 8.10.

**Comment by bolt12** [comment]
Done: https://github.com/IntersectMBO/cardano-api/pull/488 !

**Pull request closed without merging**